dts/riscv/gigadevice/gd32vf1.dtsi

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
```

Comment:
file name should likely be `gd32vf103.dtsi`

Reply:
I'll apply it.